code/__DEFINES/misc.dm

```diff
@@ -250,4 +250,4 @@
 #define MATRIX_GREYSCALE list(0.3, 0.3, 0.3, 0,\
                               0.3, 0.3, 0.3, 0,\
                               0.3, 0.3, 0.3, 0,\
-                              0,   0,   0,   1)
+                              0,   0,   0,   1)//Gun trigger guards #define TRIGGER_GUARD_ALLOW_ALL -1 #define TRIGGER_GUARD_NONE 0 #define TRIGGER_GUARD_NORMAL 1
```

Comment:
Fixed this already. Apparently TortoiseGit decided to be weird and put all the new lines on the same line when I was resolving conflicts.